docs/imagecustomizer/concepts/imagehistory.md

```diff
@@ -0,0 +1,111 @@
+# Image History
```

Comment:
Add this file to [navigation.yml](https://github.com/microsoft/azure-linux-image-tools/blob/536ee43547cbc23f0e88960474731f47ad1175c6/docs/_data/navigation.yml).